page-changelog.php

```diff
@@ -15,6 +15,108 @@
 </ul>
 
 <a name="latest10.0"></a></li>
+<h3>Version 10.0.4RC2 <small>Nov 22 2017</small><br></h3>
+<p>Release notes: <a data-cke-saved-href="https://doc.owncloud.org/server/10.0/admin_manual/release_notes.html#changes-in-10-0-3" href="https://doc.owncloud.org/server/10.0/admin_manual/release_notes.html#changes-in-10-0-3">Release notes</a><br></p>
+      <h4>Added<br></h4>
+      <ul>
+         <li>Added support for eml mimetype - <a data-cke-saved-href="https://github.com/owncloud/core/issues/29204" href="https://github.com/owncloud/core/issues/29204">#29204</a></li>
```

Comment:
looks like your magic tool has added some strange attributes `data-cke-saved-href`, can you remove them ? (let me know if you need a magic regexp)

Reply:
yes, please provide me with some regexp magic >.<
```
E486: Pattern not found: data-cke-saved-href=\"https:\/\/github.com\/owncloud\/core\/[a-z][a-z][a-z][a-z][a-z]\/[0-9][0-9][0-9][0-9][0-9]\"
```